### PR TITLE
Follow a collaborator's viewport

### PR DESCRIPTION
**Pull request by gjmooney**
Starting on #218

follow

### CODE REVIEW

File: packages/schema/src/interfaces.ts [hunk 1]
```diff
@@ -29,6 +29,10 @@ import { IRasterSource } from './_interface/rastersource';
 
 export { IGeoJSONSource } from './_interface/geojsonsource';
 
+export type CenterPosition = {
```
Comment (mfisher87) [inline packages/schema/src/interfaces.ts:32]: Nice! Hope you don't mind me leaving a comment on your draft :) 

Nit: I feel this name would work better if not for the `zoom` attribute. What about `ViewportPosition` or `ViewportState`?

File: packages/schema/src/model.ts [hunk 2]
```diff
@@ -385,6 +386,13 @@ export class JupyterGISModel implements IJupyterGISModel {
     return this._sharedModel.options;
   }
 
+  syncCenter(center?: CenterPosition, emitter?: string): void {
```
Comment (mfisher87) [inline packages/schema/src/model.ts:389]: `syncViewport`?

**Comment by gjmooney** [comment]
please update snapshots

**Review by mfisher87 — APPROVED** [review]
:star_struck:

**Review by martinRenou — CHANGES_REQUESTED** [review]
Thanks!

I'm not sure how the visual regression tests are passing.

For me the map is always opening at the same initial position, not respecting the position saved in the file anymore:

https://github.com/user-attachments/assets/bd816287-46b0-49ea-bd5a-f367b619bd53

**Comment by gjmooney** [comment]
> Thanks!
> 
> I'm not sure how the visual regression tests are passing.
> 
> For me the map is always opening at the same initial position, not respecting the position saved in the file anymore:

It looks like `view.animate` is undefined at first? So the lat/lng was getting reset to 0. I'm gonna assume the regression tests are slow enough that it wasn't an issue with them. Adding a check and `setCenter`/`setZoom` as a fallback seems to work.

**Review by martinRenou — APPROVED** [review]
Thanks!!

**Pull request merged**